kubernetes/opentelemetry-demo.yaml

```diff
@@ -10114,6 +10179,67 @@ spec:
 # Source: opentelemetry-demo/templates/component.yaml
 apiVersion: apps/v1
 kind: Deployment
+metadata:
+  name: opentelemetry-demo-imageprovider
+  labels:
+    
+    opentelemetry.io/name: opentelemetry-demo-imageprovider
+    app.kubernetes.io/instance: opentelemetry-demo
+    app.kubernetes.io/component: imageprovider
+    app.kubernetes.io/name: opentelemetry-demo-imageprovider
+    app.kubernetes.io/version: "1.10.0"
+    app.kubernetes.io/part-of: opentelemetry-demo
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      
+      opentelemetry.io/name: opentelemetry-demo-imageprovider
+  template:
+    metadata:
+      labels:
+        
+        opentelemetry.io/name: opentelemetry-demo-imageprovider
+        app.kubernetes.io/instance: opentelemetry-demo
+        app.kubernetes.io/component: imageprovider
+        app.kubernetes.io/name: opentelemetry-demo-imageprovider
+    spec:
+      serviceAccountName: opentelemetry-demo
+      containers:
+        - name: imageprovider
+          image: 'ghcr.io/open-telemetry/demo:1.10.0-imageprovider'
+          imagePullPolicy: IfNotPresent
+          ports:
+          
```

Comment:
nit: new deployment included, should we add a changelog?

Reply:
Updating this file is a lagging change for the 1.10.0 release. I'm not sure we should add a changelog entry for something captured as part of that release itself.

Reply:
This does bring up an interesting point about the post-release tasks we need to do. I could certainly see some improvements in tracking and getting those done.

Reply:
I wonder if we couldn't have a GH action that builds it and opens a PR whenever there is a new helm release. 🤔